docs/basic-features/layouts.md

```diff
@@ -0,0 +1,109 @@
+---
+description: Learn how to share components and state between Next.js pages with Layouts.
+---
+
+# Layouts
+
+<details open>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/layout-component">layout-component</a></li>
+  </ul>
+</details>
+
+The React mental model allows us to deconstruct a [page](/docs/basic-features/pages.md) into a series of components. Many of these components are often reused between pages. For example, you might have the same navigation bar and footer on every page.
```

Comment:
```suggestion
The React model allows us to deconstruct a [page](/docs/basic-features/pages.md) into a series of components. Many of these components are often reused between pages. For example, you might have the same navigation bar and footer on every page.
```

It looks like the language linting recommends we don't use this although we can add this to the allowed list if we want to keep it:

```
  14:11-14:17  warning  `mental` may be insensitive, use `rude`, `malicious`, `mean`, `disgusting`, `vile`, `person with symptoms of mental illness`, `person with mental illness`, `person with symptoms of a mental disorder`, `person with a mental disorder` instead  nuts  retext-equality
```